backend/package.json

```diff
@@ -41,6 +41,7 @@
     "pg-boss": "^8.0.0",
     "pg-format": "^1.0.4",
     "qs": "^6.10.3",
+    "seedrandom": "^3.0.5",
```

Comment:
I'm surprised the base math class didn't support seeding, weird

Reply:
Yea its frustrating. That's why I didn't go with seeds before, but I found a library that does it.